### PR TITLE
Python3 support, building on CentOS 8 and 9.

### LINKED ISSUE #4: Support python3 and CentOS 8
This should support python3, to be included in UMD for CentOS 8.

### PR DESCRIPTION
**Pull request by gwarf**
- Address errors from python and shell linters
- Add a GitHub Action worfklow to run provided test file (it maybe easier/cleaner to use something like https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python)
- Port script to python3

Close #4

### CODE REVIEW

**Comment by gwarf** [comment]
I've got some issues with the script usage block and the script:
- https://github.com/EGI-Federation/glite-info-static/blob/850881ea5c5c9fcd64f109d6377508d4821b2fd1/sbin/glite-info-static#L52 seems to imply that -c <config> is optional, while looking at https://github.com/EGI-Federation/glite-info-static/blob/850881ea5c5c9fcd64f109d6377508d4821b2fd1/sbin/glite-info-static#L116-L117, it seems required.
- https://github.com/EGI-Federation/glite-info-static/blob/850881ea5c5c9fcd64f109d6377508d4821b2fd1/sbin/glite-info-static#L58 documents that the `-c <config>` can be an absolute path, while in https://github.com/EGI-Federation/glite-info-static/blob/850881ea5c5c9fcd64f109d6377508d4821b2fd1/sbin/glite-info-static#L136 the `config_file` path is made by concatenating it to other variables, which  won't work with an absolute path :/ 

I've been quickly looking at https://github.com/EGI-Federation/glite-info-static/commit/b11ee8f2d4a75af667b98c18e1602e8b367bcd99, which seems to make think that some features may have been altered when moving from shell to python years ago.

The tests in https://github.com/EGI-Federation/glite-info-static/blob/850881ea5c5c9fcd64f109d6377508d4821b2fd1/tests/run-tests.sh#L8 are still working, and it's using a `-c` value that is not an absolute path like in the usage block.

I see two options to align things:
- ensuring that the code can do what the documentation/usage block says
- adapting the usage block to align with what the script can really do, and to what is done in the test file.

I would expect it's safer, simpler and makes more sense to do the second option and align the usage bloc to what the script can do, so basically documenting that the `-c` option is required, and ensuring it's not an absolute path.

What do people think? @andrea-manzi?

File: sbin/glite-info-static [hunk 1]
```diff
@@ -29,18 +29,22 @@
...
+
 
-# Funtion to print out the usage
 def usage():
-    sys.stderr.write('Usage: %s -m <module> [OPTIONS] \n' % (sys.argv[0]))
-    sys.stderr.write('''
-  glite-info-create.sh -m <module> [-i <ifaces>] [-t <templates>] [-c <configs>]
-                       [-p <path>] [-o <outpath>] [-d debug]
-       
+    """
```
Comment (enolfc) [inline sbin/glite-info-static:39]: Lot of code could be just removed with argparse...

**Review by enolfc — APPROVED** [review]
LGTM!

File: sbin/glite-info-static (hunk 1 above)
Comment (gwarf) [reply sbin/glite-info-static:39]: Sure, it was a lazy/time-constrained update, if you feel like you would be able to do a PR for this it would great :)

Comment (gwarf) [reply sbin/glite-info-static:39]: https://github.com/EGI-Federation/glite-info-static/pull/7

**Pull request merged**